docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/SsaDetectSpike.cs

```diff
@@ -55,7 +55,7 @@ public static void SsaSpikeDetectorTransform()
             var outputColumnName = nameof(SsaSpikePrediction.Prediction);
 
             // The transformed data.
-            var transformedData = ml.Transforms.SsaSpikeEstimator(outputColumnName, inputColumnName, 95, 8, TrainingSize, SeasonalitySize + 1).Fit(dataView).Transform(dataView);
+            var transformedData = ml.Transforms.SsaDetectSpike(outputColumnName, inputColumnName, 95, 8, TrainingSize, SeasonalitySize + 1).Fit(dataView).Transform(dataView);
```

Comment:
>SsaDetectSpike [](start = 48, length = 14)

Why not DetectSsaSpike instead? I think IID and SSA should be added to the name in the same way. right now we have DetectIidSpike and SsaDetectSpike.

Reply:
i thought of that, but that doesn't make complete sense to me.  

IID is the type of distribution. 
SSA is a method. 

`Detect Independent Identically Distributed Spike/Change `makes sense. 

and 

`Singular Spectrum Analysis Detect Spike/Change` makes sense but not `Detect Singular Spectrum Analysis Spike`


---
In reply to: [265263423](https://github.com/dotnet/machinelearning/pull/2900#discussion_r265263423) [](ancestors = 265263423)

Reply:
Just a random idea and not a required change. How about `DetectSpikeBySsa`?

---
In reply to: [265345768](https://github.com/dotnet/machinelearning/pull/2900#discussion_r265345768) [](ancestors = 265345768,265263423)

Reply:
I like that better, actually. 

---
In reply to: [265386353](https://github.com/dotnet/machinelearning/pull/2900#discussion_r265386353) [](ancestors = 265386353,265345768,265263423)